parser/src/parse.ts

```diff
@@ -22,6 +22,7 @@ import { resolve } from "path"
 import { performance } from "perf_hooks"
 import yargsParser from "yargs-parser"
 import { hydrateData } from "./hydrate"
+import path from "path/posix"
```

Comment:
is `path/posix` special?

Reply:
Not sure, this is the documentation: https://nodejs.org/api/path.html

Reply:
I think the root path module is cross platform, while `path/posix` is not